flexget/utils/imdb.py

```diff
@@ -239,10 +239,10 @@ def parse(self, imdb_id):
             return
 
         # Parse stuff from the title-overview section
-        name_elem = title_overview.find('h1', itemprop='name')
-        self.name = name_elem.find(text=True, recursive=False).strip()
+        name_elem = title_overview.find(itemprop='name')
+        self.name = name_elem.contents[0]
 
-        year = name_elem.find('a')
+        year = title_overview.find(attrs={'class': 'nobr'}).find('a', href=True)
```

Comment:
As much as we all appreciate the work on flexget, I have millions of crashreports, and I dont want crash reports, I want issue reports.. what I mean is, gracefully deal with the error and inform the user (issue report), and then move on to next task or next item in the list. if the task having this imdb error crash was first in my config, then nothing else would run...